src/common/snippets/src/lowered/loop_manager.cpp

```diff
@@ -223,6 +235,20 @@ size_t LinearIR::LoopManager::mark_loop(LinearIR::constExprIt loop_begin_pos,
     }
     return loop_id;
 }
+
+size_t LinearIR::LoopManager::mark_loop(LinearIR::constExprIt loop_begin_pos,
```

Comment:
We have already this method with `ExpressionPort`. I think we can use template method to avoid two identical methods. What do you think?

Reply:
Good idea, thanks. Template method created